compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+''''exec "python3" "onecc-docker" "$@" #'''
+
+
+
+import sys
+import subprocess
+import json
+import os
+import re
+import requests
+
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
+    out, err = p.communicate()
+    dic = {}
+
+    decoded_out = [i.split() for i in out.decode().split("\n") if len(i)]
+
+    if len(decoded_out) <2 :
+        return False
+
+    dic[decoded_out[0][0]] = decoded_out[1][0]
+    dic[decoded_out[0][1]] = decoded_out[1][1]
+
+    tag_name = cmd[2].split(':')
+
+    if dic['REPOSITORY'] == tag_name[0] and dic['TAG'] == tag_name[1]  :
+        return True 
+    else :
+        return False
+
+def write_dockerfile(script_path,version) :
+
+    with open(f'{script_path}/Dockerfile', 'w', encoding='utf-8') as dockerfile: 
+        dockerfile.write('FROM ubuntu:18.04 \n\n')
+        dockerfile.write('RUN apt update -y && \ \n')
+        dockerfile.write('\t apt install -y \ \n')
+        dockerfile.write('\t wget && \ \n')
+        dockerfile.write('\t apt clean \n')
+        dockerfile.write(f'RUN wget https://github.com/Samsung/ONE/releases/download/{version}/one-compiler_{version}_amd64.deb && \ \n')
+        dockerfile.write(f'\t apt-get install -y ./one-compiler_{version}_amd64.deb \n')
+        dockerfile.write('ENTRYPOINT ["onecc"]')
+
+
+def main():
+    onecc_version = subprocess.check_output("command -v onecc > /dev/null && echo $(onecc -v)", shell=True ,encoding='utf-8')
+    installed_onecc_version = onecc_version.split(' ')[2]
```

Comment:
What if users don't have any version of `one-compiler` installed?

Reply:
I thought that command -v onecc will return empty when onecc-version so onecc_version is empty too
but when I tested now, subprocess return nonzero exit status so It makes error on that. I'll change my code to care about absent of one-compiler.